### PR TITLE
[DOM-36776] Add repos module to rsyncd.conf

### PR DESCRIPTION
**Pull request by pocheung1**
Add repos module to rsyncd.conf to sync /repos

https://dominodatalab.atlassian.net/browse/DOM-36776

### CODE REVIEW

File: dockerfiles/rsyncd.conf [hunk 1]
```diff
@@ -1,9 +1,12 @@
 pid file = $RSYNC_RUN_DIR/rsync.pid
 lock file = $RSYNC_RUN_DIR/rsync.lock
 log file = /dev/stdout
-
-[mnt]
-path = /mnt
 use chroot = false
 read only = false
 timeout = 300
```
Comment (adp312) [inline dockerfiles/rsyncd.conf:6]: I cannot be completely sure, but at least my reading of the rsyncd.conf man page is that these 3 parameters (`use chroot`, `read only`, `timeout`) are for the modules only.  Also, this is what I saw in examples. Unless we want to go deep in testing of rsyncd, I suggest to copy them into each module individually.

**Review by adp312 — CHANGES_REQUESTED** [review]
Please copy parameters to each module.

File: dockerfiles/rsyncd.conf (hunk 1 above)
Comment (pocheung1) [reply dockerfiles/rsyncd.conf:6]: I am able to rsync to /repos with this config file.  I have also tried toggling `use chroot` and `read only` parameters to true and that causes rsync to fail.  So the global parameters are working as intended.

Comment (pocheung1) [reply dockerfiles/rsyncd.conf:6]: Here is an example here where some module parameters are configured as global:
https://linuxconfig.org/how-to-setup-the-rsync-daemon-on-linux

**Review by adp312 — APPROVED** [review]
OK then.

**Pull request merged**